docs/psf_spec/high_level_overview.rst

```diff
@@ -0,0 +1,244 @@
+properties -- position and flux primarily -- determined. For this a ``Fitter`` instance is
+required; these are drawn from a separate class or instance of minimization routines, such as
+those given under the `~astropy.modeling` umbrella. The fitter must be a two-dimensional fitter,
+by nature of the image to be fit, accepted as an input to the instance. The class must then
+return the position of the source (or sources, if a composite model was supplied; see
+`Group Maker`_) and its total flux, as determined by the PSF model describing the telescope
+system -- optics, CCD, etc. -- that created the image. See `below <PSF Model_>`_ for more
+details on the PSF model, and the :doc:`fitter <fitter>` documentation for further
+information on the ``Fitter`` instance and its properties.
+
+
+.. _PSF Model:
+
+PSF Model
+^^^^^^^^^
+Perhaps the most important aspect of PSF fitting, the ``PSF Model`` describes the distribution
+of light falling on a given pixel from a source at some position with some total flux, fully
+describing the effects of telescope optics, quantized CCD pixels, etc. In this respect it is
+more properly described as a PRF than a PSF -- see the :doc:`documentation <psf_model>` for
+more details -- but we conflate the two terms here freely. Similar to the ``Fitter``, it is
+a separate callable class detailing a specific PSF response. This can be some analytical
+function, such as `~photutils.psf.IntegratedGaussianPRF`, or an empirical description of the
+PSF, such as that implemented by the effective PSF functionality (see :ref:`build-epsf` for
+details). The model must describe a two-dimensional distribution, and must return the total
+flux of the source and its sub-pixel position via the ``evaluate`` function, accepted by the
+Fitter_ when minimizing the fit between the model and the data.
+
+
+.. _noise:
+
+Noise Description
+^^^^^^^^^^^^^^^^^
+The ``noise`` block within the PSF fitting process is, in much the same way as the ``Fitter``
+block, a separate ``callable`` function mapping the relationship between the pixel values in
+the input ``image`` and the corresponding uncertainty -- in units of standard deviations -- of
+the pixel. It must therefore accept a two-dimensional array and return an array of the same
+shape as ``image``. Alternatively it can be overloaded to indicate that the uncertainty array
+was passed along with ``image`` via an `~astropy.nddata.NDData` instance. A more detailed
+description of this block is available :doc:`here <noise_data>`.
+
+
+.. _Background Estimator:
+
+Background Estimator
+^^^^^^^^^^^^^^^^^^^^
+The ``image`` on which source extraction and evaluation is to be performed is not necessarily
+assumed to have had any pre-processing applied to it, and thus it may be necessary to account
+for the counts of the image background when handling PSF fitting. This block handles
+the determination of the background levels, avoiding high count peaks to calculate the typical
```

Comment:
what do you mean by "avoiding high count peaks"?  I think that might be algorithm-specific, so maybe better to just do this:
```suggestion
 the determination of the background levels, and should provide the typical
```
?